src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -308,19 +319,19 @@
     /**
      * Scrollinglist of the resources.
      */
-    private final ScrollingList resourceList;
+    private final ScrollingList   resourceList;
     /**
      * Inventory of the player.
      */
-    private final InventoryPlayer inventory = this.mc.player.inventory;
+    private final InventoryPlayer inventory  = this.mc.player.inventory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)